MetadataExtractor/Formats/Photoshop/DuckyReader.cs

```diff
@@ -9,14 +9,15 @@ namespace MetadataExtractor.Formats.Photoshop
     public sealed class DuckyReader : IJpegSegmentMetadataReader
     {
         public const string JpegSegmentPreamble = "Ducky";
```

Comment:
I wonder if we still need these preamble strings. Can be a future PR.

Reply:
If we're OK with a breaking change, it would be great to replace them all with the UTF8 encoded version.

Reply:
Looks like they're used in the powershell code. That could be revisited in future too.